.github/workflows/mkdocs-latest.yaml

```
@@ -7,7 +7,7 @@ on:
         required: true
   push:
     tags:
-      - "v*"
+      - "v[0-9]+.[0-9]+"
```

We need `.0` at the end, no? Otherwise, it doesn't match a version like `v0.30.0`. Please correct me if I'm wrong.

```suggestion
      - "v[0-9]+.[0-9]+.0"
```

Also, what if we remove a patch version or replace a patch version with `0` ?
- e.g. 
    - v0.30.1 => v0.30.0
    - or
    - v0.30.1 => v0.30

Let's say we have a typo in v0.30.0. We can overwrite it by pushing `v0.30.1` as the `v0.30.1` tag publishes the new `v0.30.0` document.

---

you are right my bad about forgetting `.0`. not sure I understood the second part

---

Your change no longer publishes a new documentation when a patch version update such as `v0.30.1` is pushed. But we sometimes add minor doc updates like typo in the patch version increment. I suggested pushing v0.30.1 would publish a new documentation, but as v0.30.0.

---

got it, thanks for explaining. I'll open a different PR for that